applications/incompressible_navier_stokes/vortex/application.h

```diff
@@ -258,8 +258,8 @@ class Application : public ApplicationBase<dim, Number>
 
     // TEMPORAL DISCRETIZATION
     this->param.solver_type                  = SolverType::Unsteady;
-    this->param.temporal_discretization      = TemporalDiscretization::BDFDualSplittingScheme;
-    this->param.treatment_of_convective_term = TreatmentOfConvectiveTerm::Explicit;
+    this->param.temporal_discretization      = TemporalDiscretization::BDFConsistentSplittingScheme;
+    this->param.treatment_of_convective_term = TreatmentOfConvectiveTerm::LinearlyImplicit;
```

Comment:
This is not urgent and must not happen in this PR, but would it make sense to control this an input file, such that we can run the test for several different time stepping methods (in the future)?

Reply:
defnitely, I think that would be a good way to make testing for us also faster; I often find myself changing the application setup first thing. In a follow-up, we can do this for this application as well.